<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CExportDialog</class>
 <widget class="QDialog" name="CExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Save to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_fileName"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_selectFileName">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Samplerate</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_samplerate">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>8000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11025</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22050</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>44100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>48000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>96000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_originalSamplerate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_originalSamplerate">
        <property name="text">
         <string>Original samplerate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_customSamplerate">
        <property name="text">
         <string>Custom samplerate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_changeSpeed">
        <property name="text">
         <string>Change speed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_speed">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_fileName</tabstop>
  <tabstop>toolButton_selectFileName</tabstop>
  <tabstop>radioButton_originalSamplerate</tabstop>
  <tabstop>spinBox_originalSamplerate</tabstop>
  <tabstop>radioButton_customSamplerate</tabstop>
  <tabstop>comboBox_samplerate</tabstop>
  <tabstop>checkBox_changeSpeed</tabstop>
  <tabstop>doubleSpinBox_speed</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
